### PR TITLE
Add deletion scenario to e2e test

### PR DESCRIPTION
**Pull request by 16yuki0702**
# Changes
Add deletion scenario to e2e test https://github.com/tektoncd/cli/issues/244

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -111,4 +111,42 @@ run_test  "show logs" tkn taskrun logs test-template-volume
 run_test  "list pipelineresources" tkn resource list
 run_test  "describe pipelineresource" tkn resource desc skaffold-git
 
+# delete pipeline, pipelinerun, task, taskrun, and pipelineresource
+run_test  "delete pipeline" expect -c "
+  spawn  tkn pipeline delete output-pipeline
```
Comment (16yuki0702) [inline test/e2e-tests.sh]: Is it better to be more simple using ```-f``` flag??

Comment (chmouel) [reply test/e2e-tests.sh]: I think it's fine with -f but doing with -f is extra better , you can perhaps move a lot of it to a generic shell function to simplify it,

Comment (16yuki0702) [reply test/e2e-tests.sh]: > you can perhaps move a lot of it to a generic shell function to simplify it,

thanks!! I'll try it.

**Comment by 16yuki0702** [comment]
Hmm… Is expect command not installed on e2e image?
I found the following log
```/go/src/github.com/tektoncd/cli/test/e2e-common.sh: line 202: expect: command not found```

https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_cli/279/pull-tekton-cli-integration-tests/1169157724517175296/

File: test/e2e-tests.sh (hunk 1 above)
Comment (16yuki0702) [reply test/e2e-tests.sh]: Hmm... Test conainter image seems not to have expect command so I fix to use ```-f``` flag for now.
I think we can choose another way to this like below, any ideas?

1. find a way use another command that has the same functionality as the expect command. (but I have no idea for now sorry :sweat_smile:)
2. Include expect command in test container image (Could it?)

File: test/e2e-common.sh [hunk 2]
```diff
@@ -199,7 +199,7 @@ run_test() {
   local desc="$1"; shift
 
   echo "Running $@"
-  $@ || fail_test "failed to $desc"
+  "$@" || fail_test "failed to $desc"
```
Comment (sthaha) [inline test/e2e-common.sh]: 👍

**Pull request merged**